translations/ui-finance/en.json

```diff
@@ -102,6 +102,7 @@
   "ledger.rollover.basedOn.remaining": "Remaining",
   "ledger.rollover.increaseBy": "Increase by, %",
   "ledger.rollover.closeAllCurrentBudgets": "Close all current budgets",
+  "ledger.rollover.error.conflict": "Ledger <b>{name}</b> was already rollovered from the current fiscal year (<b>{code}</b>)",
```

Comment:
Should confirm with your Product Owner, but I'm guessing this message should say something like "was already rolled over" instead of "rollovered".

Reply:
Thank you! We refined and confirm with PO error message, and your remark was correct)